### PR TITLE
Update faq.md to have updated info

### PR DESCRIPTION
**Pull request by quadsYT**
theres 2 ways to get banned. cheat online, or eShop fraud. so i updated it for eshop fraud
for the haxchi offline, indexiine is a thing that lets you use the browser exploit offline. so you dont really need haxchi
and also, makes it read that it supports the latest firmware (5.5.5)

### CODE REVIEW

File: docs/extras/faq.md [hunk 1]
```diff
@@ -1,14 +1,18 @@
 # Frequently Asked Questions {docsify-ignore-all}
 ---
+
+### Does this work on the latest version?
+Yes, at the time of reading this it's at 5.5.5
```
Comment (GaryOderNichts) [inline docs/extras/faq.md:5]: Not sure if this is really needed in the FAQ.
The Introduction page already says `This guide will work on all Wii U consoles in all regions on firmware 5.5.5 or below.`.

File: docs/extras/faq.md [hunk 2]
```diff
@@ -1,14 +1,18 @@
...
+Currently, homebrewing the Wii U does not expose you to a risk of being banned by Nintendo unless you cheat in online games, or do eShop fraud. You should, however, always be careful with launching downloaded homebrew, especially if you don't trust the source, as homebrew can potentially damage your system!
 
 ### What is Haxchi, and do I need it?
 
 Haxchi is a secondary entry point which uses certain DS Virtual Console titles to allow for access to homebrew while being completely offline (still works with online but no connection required to launch CFW)
 
-Haxchi is not required, however, if you want to use homebrew on your Wii U while offline you will need it. It’s highly recommended as it’s more stable than the browser entry point, especially on the newest firmware.
+Haxchi is not required, however, if you want to use homebrew on your Wii U without the internet browser you will need it. It’s highly recommended as it’s more stable than the browser entry point, especially on the newest firmware.
```
Comment (GaryOderNichts) [inline docs/extras/faq.md:15]: The second sentence should be changed as well as the browser exploit isn't really unstable on the latest firmware anymore.
Maybe something like `It's recommended as it's more convenient to get into CFW than using the Browser Exploit and Mocha CFW.` would work.

**Review by GaryOderNichts — CHANGES_REQUESTED** [review]

File: docs/extras/faq.md (hunk 1 above)
Comment (quadsYT) [reply docs/extras/faq.md:5]: Alright, i should probably remove it.

File: docs/extras/faq.md (hunk 2 above)
Comment (quadsYT) [reply docs/extras/faq.md:15]: Ah okay, makes sense

Comment (quadsYT) [reply docs/extras/faq.md]: alright fixed this, check out my other commit

**Comment by quadsYT** [comment]
Hey @GaryOderNichts could you check my new commit?

**Pull request merged**